cloud/app/web_components/robot-agent-device.jsx

```diff
@@ -467,22 +487,30 @@ const Device = (props) => {
           packageName="robot-agent"
         />
 
-      <Fold title="Configuration">
-        <div style={styles.row}>
-          {latestVersionData?.info?.config &&
-            <ConfigEditor info={latestVersionData.info}
-              updateConfig={
-                (modifier) => runCommand('updateConfig', {modifier}, log.debug)
-              }/>}
-        </div>
+      <Fold title="Configuration & status">
+          <>
+            <div style={styles.row}>
+              {latestVersionData?.info?.config &&
+                <ConfigEditor info={latestVersionData.info}
+                  updateConfig={
+                    (modifier) => runCommand('updateConfig', {modifier}, log.debug)
+                  }/>}
+            </div>
+            <div style={styles.row}>
+              <strong>Agent Resource Usage</strong>
+              <ResourceMetrics 
+                deviceData={latestVersionData}
+                packageName="robot-agent"
+              />
```

Comment:
Is it necessary to give the ResourceMonitor *all* of the data first and then filter it down to the named package? Or could we just give it only the data for the package we want to visualize -- which would be simpler?